zephyr/Kconfig

```diff
@@ -1,3 +1,90 @@
-# Copyright 2022 Espressif Systems (Shanghai) PTE LTD
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
 
-# This file is required by Zephyr's build system and can be empty
+if MCUBOOT
```

Comment:
I think checking for `SOC_FAMILY_ESPRESSIF_ESP32` surrounding everything might make sense. Please check.